fe/fe-core/src/main/java/org/apache/doris/analysis/DataDescription.java

```diff
@@ -809,7 +810,8 @@ private void analyzeMultiLoadColumns() throws AnalysisException {
             return;
         }
         String columnsSQL = "COLUMNS (" + columnDef + ")";
-        SqlParser parser = new SqlParser(new SqlScanner(new StringReader(columnsSQL)));
+        org.apache.doris.analysis.SqlParser parser = new org.apache.doris.analysis.SqlParser(
```

Comment:
SqlParser parser = new SqlParser(new SqlScanner(new StringReader(columnsSQL))) maybe better

Reply:
Yes, this change maybe caused by auto-formatter, I'll revert this change.